<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageRecoverWallet</class>
 <widget class="QWizardPage" name="PageRecoverWallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Select wallet type: </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radio_standard">
        <property name="text">
         <string>Standard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_viewOnly">
        <property name="text">
         <string>View-only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_multisig">
        <property name="text">
         <string>Multisig</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_hardware">
        <property name="text">
         <string>Hardware wallet</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
